### DISCUSSION TITLE
多客户端方法

### DISCUSSION THREAD

**Discussion by hujuly** [kbengine/kbengine#1565] [Q&A]
请问kbengnine是否可以支持多客户端接入同一space空间的游戏模式？比如ue4开发pc端玩家，然后html5手机网页端玩家，他们从不同的客户端类型进入同一游戏场景。

**Comment by Wandar** [comment]
可以

**Reply by hujuly** [reply]
> 可以

谢谢！可以详细一点嘛，比如3d场景的html5场景要用什么客户端的sdk？ue4和unity我看都有对应sdk，pc端我是使用ue4的客户端

**Reply by Wandar** [reply]
3d h5用unity导出就行了,或者你可以看我的仓库里有个cocos creator的demo,cocos creator可以支持3Dh5

**Reply by hujuly** [reply]
非常感谢！我先试试你仓库的，unity没接触过
